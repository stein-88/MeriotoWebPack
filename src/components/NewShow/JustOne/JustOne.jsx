import React from 'react'
import MidlePart from './MidlePart'
import FinalPart from './FinalPart'
import Fpart from './Fpart'

const JustOne = ({ ...props }) => (
        <div className="row">
            <Fpart {...props} />
            <MidlePart {...props} />
            <FinalPart {...props} />
        </div>
    )

export default JustOne
