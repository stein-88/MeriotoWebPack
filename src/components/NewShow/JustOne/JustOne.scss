.sizing {
  min-height: 65vh;
}
