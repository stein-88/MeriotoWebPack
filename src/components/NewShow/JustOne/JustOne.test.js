import React from 'react'
import renderer from 'react-test-renderer'
import JustOne from './JustOne'

describe('Test on JustOne Component', () => {
    const compJustOne = renderer.create(<JustOne />)
    test('Sould render with no props and 1 row', () => {
        const jsonJustOne = compJustOne.toJSON()
        const Result = compJustOne.root.findAllByProps({ className: 'row' })
        expect(jsonJustOne).toMatchSnapshot()
        expect(Result).toHaveLength(1)
    })
})