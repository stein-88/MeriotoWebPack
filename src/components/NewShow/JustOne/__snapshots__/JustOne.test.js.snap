// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on JustOne Component Sould render with no props and 1 row 1`] = `
<div
  className="row"
>
  <div
    className="col-4"
  >
    <section
      className="w-100"
    >
      <img
        alt=""
        className="reimg"
        src="test-file-stub"
      />
    </section>
  </div>
  <div
    className="col-4"
  />
  <div
    className="col-4"
  >
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          For company:
           
        </strong>
        no
      </small>
    </p>
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          For charity:
           
        </strong>
        no
      </small>
    </p>
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          For individual:
           
        </strong>
        no
      </small>
    </p>
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          For internal:
           
        </strong>
        no
      </small>
    </p>
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          License is required:
           
        </strong>
        no
      </small>
    </p>
    <p
      className="mb-0"
    >
      <small>
        <strong
          data-test="12elements"
        >
          <a
            rel="noreferrer"
            target="_blank"
          >
            API_URL
          </a>
           
        </strong>
      </small>
    </p>
  </div>
</div>
`;
